app/modules/asset/service.server.ts

```diff
@@ -593,16 +596,16 @@ export const getPaginatedAndFilterableAssets = async ({
   const { assets, totalAssets } = await getAssets({
     userId,
     page,
-    perPage,
+    perPage: cookie.perPage,
```

Comment:
Let's make a perPage variable and use it in all 3 places. You can just destructure it from `cookie`